packages/vscode-extension/src/common/Project.ts

```diff
@@ -145,6 +145,9 @@ export interface ProjectInterface {
   getDeepLinksHistory(): Promise<string[]>;
   openDeepLink(link: string): Promise<void>;
 
+  startRecording(): void;
+  stopRecording(): void;
+  captureRecording(): Promise<RecordingData>;
```

Comment:
can we only have one method for stopping that returns the data? What is the point of having both?

Reply:
You're right, one is enough.